lisa/schema.py

```diff
@@ -298,7 +298,7 @@ class Variable:
     )
 
     name: str = field(default="")
-    value: Union[str, bool, int, Dict[Any, Any], List[Any]] = field(default="")
+    value: Union[str, bool, int, Dict[Any, Any], List[Any], None] = field(default="")
```

Comment:
What happens if we do not specify a variable in runbook? Would it be treated as `None` after this change?